### PR TITLE
Update step-3.md to include info on updating fork

### PR DESCRIPTION
**Pull request by DavidLRowe**
Need to add an image to cdn/dev/img/developer/keyboards/ folder.

### CODE REVIEW

**Comment by mcdurdin** [comment]
Before I review, I note that the test is finding a broken link:

```
Getting links from: http://localhost:8888/developer/keyboards/github/step-3
├─BROKEN─ http://localhost:8888/developer/keyboards/github/updating-your-fork (HTTP_404)
```

(localhost:8888 is just the test host for verifying links)

File: developer/keyboards/github/step-3.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ https://github.com/YOUR-GITHUB-NAME/keyboards
 
 **Note:** The GitHub interface changes from time to time. The following images may vary from what you see on your screen.
 
+If it has been a while since you created your fork, you may see a note saying something like: 
+"This branch is 153 commits behind keymanapp:master". 
+If that's the case, you'll want to update your fork before proceeding. 
+See the [Updating your fork](updating-your-fork) section near the bottom of this page.
```
Comment (mcdurdin) [inline developer/keyboards/github/step-3.md:45]:
```suggestion
See the [Updating your fork](#updating-your-fork) section near the bottom of this page.
```

**Comment by DavidLRowe** [comment]
Thanks @mcdurdin

**Comment by mcdurdin** [comment]
Thank you for your contribution David! I love merging documentation updates from you!

**Pull request merged**